docs/usage.md

```diff
@@ -74,6 +78,43 @@ RUN mkdir -p /usr/lib/systemd/system/bootc-fetch-apply-updates.timer.d
 COPY 50-weekly.conf /usr/lib/systemd/system/bootc-fetch-apply-updates.timer.d
 ```
 
+## Air-gapped and dissconnected updates
+
+For environments without a direct connection to a centralized container
+registry, we encourage mirroring a registry in DMZ if possible or manually
+moving container images using `skopeo copy`.
+
+Systems that are completely dissconnected and require manual updates via USB
+drives this is relatively simple.
+
+Copy image to USB Drive:
+
+```skopeo copy docker://[registry]/[path to image] dir://run/media/$USER/$DRIVE/$DIR```
+
+*note, Using the dir transport will create a number of files,
```

Comment:
```suggestion
NOTE: Using the `dir` transport will create a number of files,
```

Should we point the user to https://github.com/containers/image/blob/main/docs/containers-transports.5.md?